tests/network/probes.go

```diff
@@ -227,6 +229,27 @@ var _ = SIGDescribe("[ref_id:1182]Probes", func() {
 			Entry("[test_id:TODO]with working Exec probe", createExecProbe(period, initialSeconds, timeoutSeconds, "uname", "-a"), blankIPFamily),
 		)
 
+		Context("guest agent ping", func() {
+			BeforeEach(func() {
+				vmi = libvmi.NewFedora(withMasqueradeNetworkingAndFurtherUserConfig(
+					withLivelinessProbe(createGuestAgentPingProbe(period, initialSeconds)),
+				)...)
+				vmi = tests.VMILauncherIgnoreWarnings(virtClient)(vmi)
+
+				By("Waiting for agent to connect")
+				tests.WaitAgentConnected(virtClient, vmi)
+				Expect(console.LoginToFedora(vmi)).To(Succeed())
+			})
+
+			It("[test_id:TODO] VM stops when guest agent is disabled", func() {
+				Expect(stopGuestAgent(vmi)).To(Succeed())
+
+				Eventually(func() (*v1.VirtualMachineInstance, error) {
+					return virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Get(vmi.Name, &metav1.GetOptions{})
```

Comment:
This will fail errcheck once this file is whitelisted FWIW so we may as well capture and check `err` here.